src/functions/template-tags/date.php

```diff
@@ -50,8 +52,24 @@ function tribe_format_date( $date, $display_time = true, $date_format = '' ) {
 			}
 		}
 
+		$original_locale = get_locale();
+
+		/**
+		 * Temporarily override locale to English (US) for French AM/PM time display, as French does not have a dedicated AM/PM system.
+		 */
+		if ( 'fr_FR' === $original_locale && ( 'g:i A' === $date_format || 'g:i a' === $date_format ) ) {
```

Comment:
```suggestion
		if ( 'fr_FR' === $original_locale && stripos( $date_format, 'a' ) !== false ) {
```
Don't have all the context, but should we be checking for the `a` character or just two very specific formats?

Reply:
Also, maybe just set a flag and use the flag on the two if statements so it isn't duplicated.
```
$should_convert = 'fr_FR' === $original_locale && stripos( $date_format, 'a' ) !== false;

if($should_convert) { ...
```

Reply:
It's affecting the 2 formats that use the `AM/PM` system i.e. `g:i A` and `g:i a`. 

I've updated to use a flag.